src/zui/ZUIOrganizeSidebar.tsx

```diff
@@ -1,219 +1,232 @@
 
-import ZUILogo from './ZUILogo';
-
-const drawerWidth = '5rem';
+const drawerWidth = 300;
 
 const useStyles = makeStyles((theme) => ({
-  appBar: {
-    [theme.breakpoints.up('sm')]: {
-      display: 'none',
-    },
-  },
-  content: {
-    flexGrow: 1,
-    padding: theme.spacing(3),
-  },
   drawer: {
-    [theme.breakpoints.up('sm')]: {
-      flexShrink: 0,
-      width: drawerWidth,
-    },
+    boxSizing: 'border-box',
+    flexShrink: 0,
+    transition: theme.transitions.create('width', {
+      duration: theme.transitions.duration.enteringScreen,
+      easing: theme.transitions.easing.sharp,
+    }),
+    whiteSpace: 'nowrap',
+    width: drawerWidth,
+  },
+  drawerHeader: {
+    display: 'flex',
+    justifyContent: 'flex-start',
+    padding: theme.spacing(0, 1),
+
+    // necessary for content to be below app bar
+    ...theme.mixins.toolbar,
   },
   drawerPaper: {
     display: 'none',
-    width: drawerWidth,
     [theme.breakpoints.up('sm')]: {
       display: 'block',
     },
-  },
-  menuButton: {
-    marginRight: theme.spacing(2),
-    [theme.breakpoints.up('sm')]: {
-      display: 'none',
-    },
-  },
-  root: {
-    display: 'flex',
+    overflowX: 'hidden',
+    transition: theme.transitions.create('width', {
+      duration: theme.transitions.duration.enteringScreen,
+      easing: theme.transitions.easing.sharp,
+    }),
+    width: drawerWidth,
   },
   roundButton: {
-    background: 'white',
-    borderRadius: '50%',
     height: '3rem',
     width: '3rem',
   },
-  // necessary for content to be below app bar
-  toolbar: theme.mixins.toolbar,
+  toggleDrawerPaper: {
+    [theme.breakpoints.up('sm')]: {
+      width: `calc(${theme.spacing(8)} + 1px)`,
+    },
+    transition: theme.transitions.create('width', {
+      duration: theme.transitions.duration.leavingScreen,
+      easing: theme.transitions.easing.sharp,
+    }),
+    width: `calc(${theme.spacing(7)} + 1px)`,
+  },
 }));
 
 const ZUIOrganizeSidebar = (): JSX.Element => {
+  const [mobileOpen, setMobileOpen] = useState(false);
+  const [hover, setHover] = useState(false);
+  const messages = useMessages(messageIds);
   const classes = useStyles();
   const theme = useTheme();
-  const [mobileOpen, setMobileOpen] = useState(false);
 
   const handleDrawerToggle = () => {
     setMobileOpen(!mobileOpen);
   };
 
   const router = useRouter();
-  const { orgId } = router.query as { orgId: string };
-
+  const { orgId } = useNumericRouteParams();
   const key = orgId ? router.pathname.split('[orgId]')[1] : 'organize';
 
-  const drawer = (
-    <Box
-      alignItems="center"
-      data-testid="organize-sidebar"
-      display="flex"
-      flexDirection="column"
-      height="100%"
-      justifyContent="space-between"
-    >
-      <List disablePadding>
-        <Box display="flex" flexDirection="column">
-          <ListItem disableGutters>
+  const [open, setOpen] = useState(false);
+  const toggleDrawer = () => {
+    setOpen(!open);
+  };
```

Comment:
This doesn't seem to be doing anything.

Reply:
I think I need it to set the the open conditional variable, but maybe it's a good idea to change the name because otherwise it's confusing